### ISSUE TITLE
Is there a way to merge chunks?

### ISSUE THREAD

**Issue by Karlheinzniebuhr** [Olegas/node-chunking-streams#10]
I'm splitting files into multiple chunks, is there a way to put them back together?

**Comment by kylefarris** [comment]
Where are you planning on reassembling them? In another file? If so, the usual answer would be to write them to a filestream.

**Comment by Olegas** [comment]
Yes, as @kylefarris said, just write them to any other stream of your choice.
